test/built-ins/Promise/executor-function-name.js

```diff
@@ -22,3 +23,6 @@ function NotPromise(executor) {
 Promise.resolve.call(NotPromise);
 
 assert.sameValue(Object.prototype.hasOwnProperty.call(executorFunction, "name"), false);
+assert.sameValue(Object.getOwnPropertyNames(executorFunction).join(), "length");
```

Comment:
this seems a bit out of scope for this test. This is asserting the only own string-based property names in executorFunction is `"length"` right after you already check that `name` is not an own property of `executorFunction`.

Reply:
My understanding here is that `Object.getOwnPropertyNames` is used in original test file because it is a stress test and `Object#hasOwnProperty` in hot loop may give incorrect results.

Stress tests are surely out of scope.

71e1291